src/IntegrationTests/Applications/SampleAspNetCoreApp/appsettings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+
+  "AllowedHosts": "*",
+
+  "NewRelic": {
+    "ApiKey": "{YOUR_API_KEY}",
```

Comment:
Does this cause a problem?   If we are using environment variables here, would a local file override the environment variable value?

Reply:
the environment variable config will override this if it exists. I have tested.